_data/riot_boards.yml

```diff
@@ -0,0 +1,416 @@
+- name: Arduino Duemilanove
```

Comment:
This file is generated during deployment as far as I can tell, so should it really be added to the source?